config/manifests/kustomization.yaml

```diff
@@ -2,4 +2,4 @@
 resources:
   - ../default
   - ../samples
-  - ../scorecard
+  #- ../scorecard
```

Comment:
Could you clarify, in the commit message, why scorecard should be disabled?

Reply:
Because we are still not using it in our CI
But you know what I'll keep it and I'll add a validation workflow in our next release